spy/build/config.py

```diff
@@ -21,40 +22,32 @@ class BuildConfig:
     "--std=c99",
     "-Werror=implicit-function-declaration",
     "-Wfatal-errors",
-    "-fdiagnostics-color=always", # force colors
-    "-I", str(spy.libspy.INCLUDE),
+    "-fdiagnostics-color=always",  # force colors
+    "-I",
+    str(spy.libspy.INCLUDE),
 ]
-RELEASE_CFLAGS  = ["-DSPY_RELEASE", "-O3", "-flto"]
+RELEASE_CFLAGS = ["-DSPY_RELEASE", "-O3", "-flto"]
 RELEASE_LDFLAGS = ["-flto"]
 
-DEBUG_CFLAGS    = ["-DSPY_DEBUG",   "-O0", "-g"]
+DEBUG_CFLAGS = ["-DSPY_DEBUG", "-O0", "-g"]
```

Comment:
all the `*_CFLAGS` variables were supposed to be aligned so that it's easier to visually see the diffs between them